inspect_builder_test.go

```diff
@@ -148,14 +290,16 @@ func testInspectBuilder(t *testing.T, when spec.G, it spec.S) {
 					})
 
 					when("the image has no mixins", func() {
-						it.Before(func() {
-							h.AssertNil(t, builderImage.SetLabel("io.buildpacks.stack.mixins", ""))
-						})
+						when("no depth argument is given", func() {
```

Comment:
I'm a bit confused - where does `depth` enter into this test? It doesn't seem relevant to `inspect_builder[_test].go`, it seems to only relate to `internal/commands/inspect_builder`

Reply:
Haha, yep lets just remove this `when` block.